e2e/scenario/base_vhd_catalog.json

```diff
@@ -13,6 +13,10 @@
         "gen2containerd": {
             "artifactName": "2204-gen2-containerd",
             "resourceId": "/subscriptions/8ecadfc9-d1a3-4ea4-b844-0d9f87e4d7c8/resourceGroups/aksvhdtestbuildrg/providers/Microsoft.Compute/galleries/PackerSigGalleryEastUS/images/2204Gen2/versions/1.1699020621.21398"
+        },
+        "gen2containerdprivatekubepkg": {
+            "artifactName": "2204-gen2-containerd",
```

Comment:
we can remove the artifact name here since we'll just opt towards using the pinned version for this scenario in all cases